x-pack/.i18nrc.json

```diff
@@ -38,7 +38,8 @@
     "xpack.transform": "legacy/plugins/transform",
     "xpack.upgradeAssistant": "legacy/plugins/upgrade_assistant",
     "xpack.uptime": "legacy/plugins/uptime",
-    "xpack.watcher": "legacy/plugins/watcher"
+    "xpack.watcher": "legacy/plugins/watcher",
+    "xpack.endpoint": "plugins/endpoint"
```

Comment:
it appears we've been trying, with some success (*cough* logstash *cough*), to keep this list in alphabetical order.  move up?

Reply:
Is there a reason for keeping it in that order?